### PR TITLE
Document console featured applications

### PR DESCRIPTION
**Pull request by spadgett**
@jwforres @ahardin-rh

### CODE REVIEW

**Review by jwforres — APPROVED** [review]

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -587,6 +587,63 @@ endif::[]
...
+----
+ifdef::openshift-origin[]
+# systemctl restart origin-master
+endif::[]
+ifdef::openshift-enterprise[]
+# systemctl restart atomic-openshift-master
+endif::[]
+----
+
+endif::[]
```
Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: Is this a rogue endif::[] ?

**Comment by ahardin-rh** [comment]
This looks great. Thanks! I just had one question.

File: install_config/web_console_customization.adoc (hunk 1 above)
Comment (spadgett) [reply install_config/web_console_customization.adoc]: Yeah, looks like it. Removed.

**Comment by spadgett** [comment]
@ahardin-rh Updated, thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Added a section on xref:../install_config/web_console_customization.adoc#configuring-featured-applications[Configuring Featured Applications].
%

**Pull request merged**